src/GameLoop.tsx

```diff
@@ -103,6 +103,39 @@ const timestamps = {
     playerInstantDropAnimDurationMilliseconds: 0,
 };
 
+const playerCellsInit = generateUserCells();
+const initialState = {
+    boardCellMatrix: createBoard(BOARD_ROWS, BOARD_COLS),
+    playerPos:[...spawnPos] as const,
+    playerCells: generateUserCells(),
+    playerAdjustedCells: convertCellsToAdjusted(playerCellsInit, [...spawnPos] as const),
+    playerFallingLettersBeforeAndAfter: [],
+    fallingLettersBeforeAndAfter: [],
+};
+
+const reducer = (state, action) => {
```

Comment:
Please type this.  You probably won't need a default case if you type it correctly (as a discriminated union).

Reply:
Done. Do you have any suggested improvements for brevity?